chia/rpc/wallet_rpc_api.py

```diff
@@ -1727,6 +1727,66 @@ async def nft_set_nft_did(self, request):
         spend_bundle = await nft_wallet.set_nft_did(nft_coin_info, did_id, fee=fee)
         return {"wallet_id": wallet_id, "success": True, "spend_bundle": spend_bundle}
 
+    async def nft_set_bulk_nft_did(self, request):
```

Comment:
I won't hold up the review, but my only comments are related to documenting the params and validating those params earlier with a useful error message to the caller. If, for example, the DID isn't set, or `nft_coin_list` is missing, or some nested param is missing, the caller isn't likely to get a very useful error message.